vaapi/vaapiimagepool.cpp

```diff
@@ -37,7 +37,7 @@ ImagePoolPtr VaapiImagePool::create(const DisplayPtr& display, uint32_t format,
     ImagePoolPtr pool;
     std::vector<ImagePtr> images;
     images.reserve(count);
-    for (size_t i = 0; i < count; ++i) {
+    for (int32_t i = 0; i < count; ++i) {
```

Comment:
This one really ought to be size_t... and "count" should be changed to size_t instead  (which means IMAGE_POOL_SIZE should be a size_t in vaapidecsurfacepool.cpp.